spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```

Comment:
- [ ] Some tests are failing . Please make sure to fix them and make all the test passed as required for this project :+1: 
![Screenshot_20230208_204912](https://user-images.githubusercontent.com/6615394/217648123-386203d1-9afb-426e-a0f3-e9f17204394a.png)